src/pipecat/services/deepgram.py

```diff
@@ -147,21 +148,15 @@ def __init__(
 
     @property
     def vad_enabled(self):
-        return self._live_options.vad_events
+        return self._settings["live_options"].vad_events
 
     def can_generate_metrics(self) -> bool:
         return self.vad_enabled
 
     async def set_model(self, model: str):
         await super().set_model(model)
         logger.debug(f"Switching STT model to: [{model}]")
-        self._live_options.model = model
-        await self._disconnect()
-        await self._connect()
-
-    async def set_language(self, language: Language):
```

Comment:
similar to elevenlabs. i think this needs to preserve the disconnect/connect behavior